doc/langref.html.in

```diff
@@ -9954,6 +9953,164 @@ const c = @cImport({
+          clang's <code>-D</code> flag</a>.
+        </li>
+        <li>
+          <code class="shell">-cflags [flags] --</code>: Pass arbitrary additional
+          <a href="https://releases.llvm.org/12.0.0/tools/clang/docs/ClangCommandLineReference.html">command line
+          flags</a> to clang. Note: the list of flags must end with <code>--</code>
+        </li>
+        <li>
+          <code class="shell">-target</code>: The {#link|target triple|Targets#} for the translated Zig code.
+          If no target is specified, the current host target will be used.
+        </li>
+      </ul>
+      {#header_close#}
+      {#header_open|Using -target and -cflags#}
+      <p>
+        <strong>Important!</strong> When translating C code with <code class="shell">zig translate-c</code>,
+        you <strong>must</strong> use the same <code>-target</code> triple that you will use when compiling
+        the translated code. In addition, you <strong>must</strong> ensure that the <code>-cflags</code> used,
+        if any, match the cflags used by code on the target system. Using the incorrect <code>-target</code>
+        or <code>-cflags</code> could result in clang or Zig parse failures, or subtle ABI incompatibilities
+        when linking with C code.
+      </p>
+      <p class="file">varytarget.h</p>
+      <pre><code class="c">long FOO = __LONG_MAX__;</code></pre>
+      <pre><code class="shell">$ zig translate-c -target <strong>thumb-freestanding-gnueabihf</strong> varytarget.h|grep FOO
+pub export var FOO: c_long = <strong>2147483647</strong>;</code></pre>
+      <pre><code class="shell">$ zig translate-c -target <strong>x86_64-macos-gnu</strong> varytarget.h|grep FOO
+pub export var FOO: c_long = <strong>9223372036854775807</strong>;</code></pre>
+      <p class="file">varycflags.h</p>
+      <pre><code class="c">enum FOO { BAR };
+int do_something(enum FOO foo);</code></pre>
+      <pre><code class="shell">$ zig translate-c varycflags.h|grep -B1 do_something
+pub const enum_FOO = <strong>c_uint</strong>;
+pub extern fn do_something(foo: enum_FOO) c_int;</code></pre>
+      <pre><code class="shell">$ zig translate-c <strong>-cflags -fshort-enums --</strong> varycflags.h|grep -B1 do_something
+pub const enum_FOO = <strong>u8</strong>;
+pub extern fn do_something(foo: enum_FOO) c_int;</code></pre>
+      {#header_close#}
+      {#header_open|@cImport vs translate-c#}
+      <p>{#syntax#}@cImport{#endsyntax#} and <code class="shell">zig translate-c</code> use the same underlying
+      C translation functionality, so on a technical level they are equivalent. In practice,
+      {#syntax#}@cImport{#endsyntax#} is useful as a way to quickly and easily access numeric constants, typedefs,
+      and record types without needing any extra setup. For more complex translation tasks such as
+      function-like macros and C functions, or if you need to pass {#link|cflags|Using -target and -cflags#} to clang,
```

Comment:
What's this about? Function-like macros work perfectly fine with `@cImport` too.

Reply:
Ah, I meant that it allows you to edit them - for function-like macros adding type info instead of `anytype`; for regular functions changing `[*c]T` to the relevant pointer type and/or making the functions look more like canonical Zig. I can clarify or just drop that part if you'd like.